roles/docker/templates/etc/apt/sources.list.d/docker.list

```diff
@@ -0,0 +1 @@
+deb https://apt.dockerproject.org/repo ubuntu-trusty main
```

Comment:
What drops this in place?  Don't you already have a apt_repository statement for something like this in tasks/main.yml?